.github/workflows/test.yml

```diff
@@ -51,29 +51,48 @@ jobs:
       - name: Run tests
         run: cargo test -- --nocapture --test-threads=1
 
-  # test_windows:
-    # name: Run tests (Windows)
-    # runs-on: windows-latest
-    # strategy:
-      # matrix:
-        # rust-version: [stable-gnu]
-    # steps:
-      # - uses: actions/checkout@v2
-      # - uses: actions-rs/toolchain@v1
-        # with:
-          # toolchain: ${{ matrix.rust-version }}
-          # target: x86_64-pc-windows-gnu
-      # - uses: msys2/setup-msys2@v2
-      # - name: Install dependencies
-        # shell: msys2 {0}
-        # run: |
-          # pacman -S --noconfirm mingw-w64-x86_64-clang mingw-w64-x86_64-toolchain
-      # - uses: r-lib/actions/setup-r@v1
-      # - name: Build
-        # run: msys2 -c 'C:/Rust/.cargo/bin/cargo build'
-        # env:
-          # LIBCLANG_PATH: C:\msys64\mingw64\bin
+  test_windows:
+    name: Run tests (Windows)
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
+    strategy:
+      matrix:
+        rust-version: [stable-gnu]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: msys2/setup-msys2@v2
+        with:
+          msystem: MINGW64
+          path-type: inherit
+          release: false
+          update: false
+          install: >-
+            mingw-w64-x86_64-clang
+            mingw-w64-x86_64-headers-git
+            mingw-w64-x86_64-make
+            mingw-w64-x86_64-pkg-config
+      - uses: r-lib/actions/setup-r@v1
+        with:
+          r-version: 'release'
```

Comment:
Can you replicate the R setup you're using here (with `r-lib/actions/setup-r@v1`) to the other operating systems? I think it's preferable to just installing `r-base-dev` because it allows us to specify the R version we're testing. (And ultimately, we should have some tests on `release`, some on `devel`, and maybe some on the previous release.)